docs/03_tutorials/index.md

```diff
@@ -3,7 +3,6 @@ title: Tutorials
 sidebar_position: 4
 ---
 
-1. [BIOS Boot Sequence](10_bios-boot-sequence.md): Demonstrates how to bootstrap a new Antelope blockchain.
-2. Tic-tac-toe Game Contract: Build a simple tic-tac-toe game on Antelope to apply the knowledge you acquired earlier. Two versions are available:
-   - [Tic-tac-toe using local single node](20_tic-tac-toe-game-smart-contract-single-node.md)
-3. [Elemental Battles](https://battles.eos.io?utm_source=devportal): Created for developers new to blockchain, Elemental Battles is part-tutorial, part-game development. You’ll play in a fantasy world and harness the powers of wood, water, and fire to learn blockchain basics and build your first game on Antelope.
+1. [BIOS Boot Sequence](10_bios-boot-sequence.md): Demonstrates how to bootstrap a new Antelope blockchain
+2. [Tic-tac-toe Game Contract](20_tic-tac-toe-game-smart-contract-single-node.md): Build a simple tic-tac-toe game on Antelope to apply the knowledge you acquired earlier
+3. [BIOS Boot Tutorial](bios-boot-tutorial.md): The bios-boot-tutorial.py script simulates the bios boot sequence
```

Comment:
this file is copied from the leap repo by the devdocs scripts before the docs are generated